index.js

```diff
@@ -40,27 +40,31 @@ function initialize(options) {
     sdkKey,
     datafile,
     logLevel,
-    datafileOptions,
   } = options;
 
 
   const defaultLogger = require('@optimizely/optimizely-sdk').logging;
-  const manager = new DatafileManager({
-    sdkKey,
+
+  const optimizelyClient = OptimizelySdk.createInstance({
+    logger: defaultLogger.createLogger({
+      logLevel: logLevel
+    }),
     ...options,
-    ...datafileOptions,
   });
 
   function updateDatafile() {
     datafile = manager.get()
     datafile._lastUpdated = new Date();
-    console.log('[Optimizely] Datafile Updated!');
+    console.log('[OPTIMIZELY] Datafile Updated!');
   }
 
-  manager.on('update', updateDatafile);
-  manager.onReady().then(updateDatafile);
-
-  manager.start();
+  const manager = optimizelyClient.projectConfigManager.datafileManager;
+  if (manager) {
```

Comment:
This can be null, like if the SDK is misconfigured (for example, this was null in CI for us when testing out this branch because we don't load Optimizely in CI). I think a warning is sufficient to cover this case.